renpy/common/00action_menu.rpy

```diff
@@ -133,6 +133,44 @@ init -1500 python:
                 return eval(config.show_menu_enable[screen])
             else:
                 return True
+                
+    @renpy.pure
+    class Continue(Action, DictEquality):
+        """
+        :doc: menu_action
+        
+        Causes the last save to be loaded.
+        The purpose of this is to load the player's last save 
+        from the main menu.
+        
+        `regexp`
+            If present, will be used in `renpy.newest_slot`. 
+            By default this searches for all manually saves.
+        """
+
+        def __init__(self, regexp=r"\d+"):
+            self.regexp = regexp
+        def __call__(self):
+            if not self.get_sensitive():
+                return
```

Comment:
I don't think sensitive should be added here as devs can add `sensitive` property to the widgets